styles.css

```diff
@@ -1,57 +1,57 @@
-body{
-    width: 375px;
-    align-items: center;
+body {
+  width: 375px;
+  align-items: center;
 }
 
-nav{
+nav {
   background-color: white;
-  margin: 10px
+  margin: 10px;
 }
 
-.header1{
-  display:flex;
+.header1 {
+  display: flex;
   justify-content: space-between;
 }
 
-.header2{
-  display:flex;
+.header2 {
+  display: flex;
   justify-content: space-between;
 }
 
-.menu-button{
+.menu-button {
   width: 40px;
   padding-right: 30px;
   height: 50px;
 }
 
-header{
+header {
   text-align: center;
-  background-image: url(../images/header-illsutration-mobile@2x.png)
+  background-image: url(../images/header-illsutration-mobile.svg);
 }
```

Comment:
- [ ]  Please make sure that the container of the section is a `section` tag.
   * It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

- [ ]  Kindly make sure that the section has the proper background image (decorations are always background images, not `img` tags).
   * If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

### Your design 👇
![stback](https://user-images.githubusercontent.com/66992901/198010328-490e753f-6eff-4b54-8e5c-f1f295e1244f.PNG)

### template design👇
![back](https://user-images.githubusercontent.com/66992901/198010378-364d9b23-b186-40fd-9055-5d38b6af4c12.PNG)

Reply:
Corrections made. Please if you think I am still getting the image input wrong, please kindly give me more information to better understand what I am required to do. Thank you for your feedback